src/plonk.rs

```diff
@@ -38,6 +39,7 @@ pub struct ProvingKey<C: CurveAffine> {
     vk: VerifyingKey<C>,
     // TODO: get rid of this?
     l0: Polynomial<C::Scalar, ExtendedLagrangeCoeff>,
+    fixed_values: Vec<Polynomial<C::Scalar, LagrangeCoeff>>,
```

Comment:
Let's make an issue for removing entries in this vector that we don't actually use because they aren't involved in a lookup argument.

Reply:
Opened https://github.com/zcash/halo2/issues/57.